src/Mock/getMock.js

```diff
@@ -93,7 +94,8 @@ export const getCollectionMock = {
     }
   },
   getOnlyThree: {
-    path: 'tests/get/users?limit=3',
+    path: 'tests/get/users',
+    body: { limit: 3 },
```

Comment:
don't we know what happens when something along the lines of `{limit : -1}`, `{limit : 'lalalalala'}` or some other invalid value for the limit is passed?

Reply:
I think firebase will throw an error. But you are right. It is a very good idea to add a test with this failure case :+1: